frontend/src/js/common/helpers/fileHelper.ts

```diff
@@ -1,8 +1,11 @@
 export const readFileAsText = (file: File) =>
-  new Promise((resolve, reject) => {
+  new Promise<string>((resolve, reject) => {
     const reader = new FileReader();
 
-    reader.onload = (evt) => resolve(evt.target.result);
+    reader.onload = (evt) =>
+      evt.target
```

Comment:
Sometimes you use exists and sometimes not, is there an explicit pattern behind that?

Reply:
Yes, `exists` is mostly used as a type guard. In this case it's not needed, ternary auto-casts into boolean